src-tauri/src/main.rs

```diff
@@ -84,17 +84,15 @@ fn main() {
 
             store.load().unwrap_or_default();
 
-            // let first_run = true;
-            let first_run = store
-                .get("first_run".to_string())
-                .unwrap_or(&serde_json::Value::Bool(true))
-                .as_bool()
-                .unwrap();
-
-            let recording_permission: bool = scap::has_permission();
+            let first_run = true;
```

Comment:
Is this change expected?

Reply:
yes, the has_permission was preventing final (prod) builds from even opening properly on mac 😅
have removed it from the code, but on my list to investigate today: https://discord.com/channels/1044321624545046569/1194965312676827148/1232288042203611136